BE/docker/Dockerfile

```
@@ -0,0 +1,33 @@
+# Backend Dockerfile for Spring Boot application
+# Multi-platform support (AMD64, ARM64)
+FROM eclipse-temurin:21-jdk AS build
```

We're compiling openjdk 24
```suggestion
FROM openjdk:24-jdk AS build
```

---

changed